test/test_head_request.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding; utf-8 -*-
+"""
+test_head_request
+~~~~~~~~~~~~~~~~~
+"""
+import h2.connection
+import pytest
+
+
+class TestHeadRequest(object):
+
+        example_request_headers = [
+            (u':authority', u'example.com'),
+            (u':path', u'/'),
+            (u':scheme', u'https'),
+            (u':method', u'HEAD'),
+            (u':content_length', u'1'),
```

Comment:
Ah, sorry: this test is not quite right.

Firstly, the `content-length` header should be in the *response*, not the request. Secondly, the header name is `content-length`, not `:content-length`.